test/commands/lib/actions-watcher.test.js

```diff
@@ -176,10 +178,10 @@ test('onChange handler calls buildActions without filterActions when actions are
   await actionsWatcher({ config: cloneApplication, log })
   expect(typeof onChangeHandler).toEqual('function')
 
-  deployActions.mockImplementation(async () => await sleep(2000))
-  onChangeHandler('/myactions/action.js')
+  buildAndDeploy.mockImplementation(async () => await sleep(2000))
+  onChangeHandler('/myactions/utils.js')
```

Comment:
Do we only watch action source file or action folder?
From issue description I felt if files outside action source folder are modifies then we ask for restart. is this correct?

Reply:
@sandeep-paliwal 
We watch the action folder and deploy only when an action file source gets changed. 
When a file in the action folder, other than the action input, gets changed, we notify the user that a restart is required in order to redeploy the actions affected by that file.